lib/Dialect/TorchConversion/Transforms/BackendTypeConversionPasses.cpp

```diff
@@ -151,6 +160,9 @@ struct FinalizingBackendTypeConversionPass
 
     if (failed(applyFullConversion(func, target, std::move(patterns))))
       signalPassFailure();
+
+    // Drop attributes that are no longer used after conversion out of Torch.
+    stripTorchAttrs(func);
```

Comment:
Are there users of torch-mlir that would miss these attributes after conversion? I could make this optional (via a separate pass, an option here, etc.)

Reply:
I don't think so. Historically there haven't been any/many torch dialect attributes used. I suspect that we just added some and forgot to make the conversion "full".